go/vt/wrangler/traffic_switcher.go

```diff
@@ -1772,23 +1772,28 @@ func getRenameFileName(tableName string) string {
 func (ts *trafficSwitcher) removeSourceTables(ctx context.Context, removalType workflow.TableRemovalType) error {
 	err := ts.ForAllSources(func(source *workflow.MigrationSource) error {
 		for _, tableName := range ts.Tables() {
-			query := fmt.Sprintf("drop table %s.%s",
-				sqlescape.EscapeID(sqlescape.UnescapeID(source.GetPrimary().DbName())),
-				sqlescape.EscapeID(sqlescape.UnescapeID(tableName)))
+			PrimaryDbName, err := sqlescape.EnsureEscaped(source.GetPrimary().DbName())
```

Comment:
Seems off to have `primaryDbName` with a capital `P` here, that isn't afaik right Go variable naming style.